requirements/test.txt

```diff
@@ -1,8 +1,6 @@
 flake8
-mock
 pytest
-coverage
 codecov
-requests-mock
-pytest-mock
```

Comment:
most likely we will need both `pytest-mock` and `requests-mock` as well as `pytest-cov` 

since the CI isn't configured, perhaps it's safer to add these in and remove later.